juris-be.csl

```diff
@@ -14,7 +14,7 @@
     <summary>Juris-M style module for Belgium</summary>
     <updated>2013-01-26T22:06:38+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
-    <law-module types="legislation"/>
+    <law-module types="legal_case legislation"/>
```

Comment:
The changes submitted so far do not provide any code to render `legal_case` items, so enabling that type on the module would just yield a CSL ERROR message instead of a citation. Do you plan to fill out the rest of the style, or should I drop this pull request?